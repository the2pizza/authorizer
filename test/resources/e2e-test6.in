{"account": {"active-card": true, "available-limit": 100}}
{"transaction": {"merchant": "Habbib's", "amount": 5, "time": "2019-02-13T12:02:00.000Z"}}
{"transaction": {"merchant": "KFC", "amount": 15, "time": "2019-02-13T12:02:05.000Z"}}
{"transaction": {"merchant": "Habbib's", "amount": 25, "time": "2019-02-13T12:02:07.000Z"}}
{"transaction": {"merchant": "Habbib's", "amount": 10, "time": "2019-02-13T12:02:09.000Z"}}