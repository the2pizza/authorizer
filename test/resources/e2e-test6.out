{"account":{"active-card":true,"available-limit":100},"violations":[]}
{"account":{"active-card":true,"available-limit":95},"violations":[]}
{"account":{"active-card":true,"available-limit":80},"violations":[]}
{"account":{"active-card":true,"available-limit":55},"violations":[]}
{"account":{"active-card":true,"available-limit":55},"violations":["high-frequency-small-interval"]}